doc/conf_ref.rst

```diff
@@ -264,6 +272,10 @@ Right side of every repo option can be enriched by the following variables:
     Refers to the base architecture of the system. For example, i686 and i586 machines
     both have a base architecture of i386, and AMD64 and Intel64 machines have a base architecture of x86_64.
 
+``$releasever``
+
+    Refers to the release version of operating system which DNF derives from information available in RPMDB.
```

Comment:
That's not actually true (at least it will not be true with libhif). It takes `VERSION_ID` from [os-release](https://www.freedesktop.org/software/systemd/man/os-release.html). Which is only one correct way.